timescaledb/overview/release-notes/index.md

```diff
@@ -26,27 +28,28 @@ If you have questions about distributed hypertables, join our #multinode channel
 on [community Slack](https://slack.timescale.com/) for installation details and
 follow these [setup instructions][distributed-hypertables-setup].
 
-### What's new in TimescaleDB 2.5:
+### What's new in TimescaleDB 2.6:
 
-- Continuous aggregates for distributed hypertables
-- Support for PostgreSQL 14
-- Experimental: Support for timezones in `time_bucket_ng`, including the
-  `origin` argument
+- Continuous aggregates with compression
+- `time_bucket_ng` support for N months and Timezones on continuous aggregates
```

Comment:
We also support years according to https://github.com/timescale/timescaledb/pull/3753

Reply:
We support N months, years, n months and years... I believe it confuses the messaging.